mkdocs_monorepo_plugin/helpers.py

```diff
@@ -0,0 +1,89 @@
+from os import path, sep
+
+def normalize(path):
+  normalized = path
+  if path.startswith('/'):
+    normalized = normalized[1:]
+  if not path.endswith('/'):
+    normalized = '{}/'.format(path)
+  return normalized
+
+def get_root_config_dir_head(config, plugin):
+  abs_root_config_file_dir = path.dirname(config['config_file_path'])
+  return path.relpath(plugin.originalDocsDir, abs_root_config_file_dir)
+
+def get_page_config_dir_head(config, page):
+  abs_root_config_file_dir = path.dirname(config['config_file_path'])
+  rel_page_markdown_file_path = path.relpath(page.file.abs_src_path, abs_root_config_file_dir)
+  return rel_page_markdown_file_path.split(sep)[0]
+
+def get_page_config_file_dir(config, page):
+  abs_root_config_file_dir = path.dirname(config['config_file_path'])
+  page_config_dir_head = get_page_config_dir_head(config, page)
+  return path.join(abs_root_config_file_dir, page_config_dir_head)
+
+def get_page_config_file_path(config, page):
+  abs_page_config_file_dir = get_page_config_file_dir(config, page)
+  abs_page_config_file_path = path.join(abs_page_config_file_dir, 'mkdocs.yaml')
+  if not path.exists(abs_page_config_file_path):
+    abs_page_config_file_path = path.join(abs_page_config_file_dir, 'mkdocs.yml')
+  return abs_page_config_file_path
+
+def get_page_config_file_yaml(config, page):
+  abs_page_config_file_path = get_page_config_file_path(config, page)
+  with open(abs_page_config_file_path, 'rb') as f:
+    return { 'docs_dir': 'docs', **yaml_load(f) }
+
+def replace_page_repo_url(config, page):
+  page_config_file_yaml = get_page_config_file_yaml(config, page)
+  if 'repo_url' in page_config_file_yaml:
+    page_config_repo_url = normalize(page_config_file_yaml['repo_url'])
+    page.edit_url = page.edit_url.replace(config['repo_url'], page_config_repo_url)
+
+def replace_page_docs_dir(config, page, plugin):
+  root_config_dir_head = get_root_config_dir_head(config, plugin)
+  page_config_dir_head = get_page_config_dir_head(config, page)
+  page_config_file_yaml = get_page_config_file_yaml(config, page)
+  page_config_docs_dir = page_config_file_yaml['docs_dir']
+
+  root_docs_dir_text = '/{}/'.format(root_config_dir_head)
+  page_docs_dir_text = '/{}/{}/'.format(page_config_dir_head, page_config_docs_dir)
+
+  page.edit_url = page.edit_url.replace(root_docs_dir_text, page_docs_dir_text)
+
+def replace_page_edit_uri(config, page, plugin):
+  page_config_file_yaml = get_page_config_file_yaml(config, page)
+  if 'edit_uri' in page_config_file_yaml:
+    page_config_edit_uri = normalize(page_config_file_yaml['edit_uri'])
+    page.edit_url = page.edit_url.replace(config['edit_uri'], page_config_edit_uri)
+  else:
+    replace_page_docs_dir(config, page, plugin)
+
+def replace_page_dir_alias(page):
+  page_docs_dir_alias = page.url.split('/')[0]
+  page_docs_dir_alias_text = '/{}'.format(page_docs_dir_alias)
+  page.edit_url = page.edit_url.replace(page_docs_dir_alias_text, '')
+
+def is_index(page):
```

Comment:
I'm wondering if this should be named and represent the logic of `is_root` instead of `is_index` (or something similar).

The problem I'm seeing: If I add a second page in the top-level `mkdocs.yml` file, I get a `FileNotFoundError` on `/docs/mkdocs.yml`.

The diff that triggers it:

```diff
diff --git a/__tests__/integration/fixtures/bug-include-path-edit-uri/docs/other.md b/__tests__/integration/fixtures/bug-include-path-edit-uri/docs/other.md
new file mode 100644
index 0000000..c78592f
--- /dev/null
+++ b/__tests__/integration/fixtures/bug-include-path-edit-uri/docs/other.md
@@ -0,0 +1,3 @@
+# Top Level Project Documentation - Other
+
+Additional information about the top-level project.
diff --git a/__tests__/integration/fixtures/bug-include-path-edit-uri/mkdocs.yml b/__tests__/integration/fixtures/bug-include-path-edit-uri/mkdocs.yml
index 0a4f69b..08a744a 100644
--- a/__tests__/integration/fixtures/bug-include-path-edit-uri/mkdocs.yml
+++ b/__tests__/integration/fixtures/bug-include-path-edit-uri/mkdocs.yml
@@ -8,4 +8,5 @@ plugins:
 
 nav:
   - Home: "index.md"
+  - Other: "other.md"
   - API: "!include api/mkdocs.yml"
```